Multisig/UI/App/RemoteNotificationHandler.swift

```diff
@@ -168,8 +168,8 @@ class RemoteNotificationHandler {
             let appConfig = App.configuration.app
             var timestamp: String?
             if let _ = try? App.shared.keychainService.data(forKey: KeychainKey.ownerPrivateKey.rawValue) {
-                // add timestamp if there is a signing key
-                timestamp = String(Int(Date().timeIntervalSince1970 * 1_000))
+                // add timestamp in seconds if there is a signing key
+                timestamp = String(Int(Date().timeIntervalSince1970))
```

Comment:
I think that potentially the Int(Double) can crash because of the big value of Double (scientific notation) not fitting into Int. It is better to format the Double as a value  without decimal point

Reply:
agree